### ISSUE TITLE
Unable to install with node 8.9.0

### ISSUE THREAD

**Issue by zfoster** [bustle/aws-sudo#9]
```
yarn global add aws-sudo

{ Error: incorrect header check
    at Gunzip.zlibOnError (zlib.js:153:15) errno: -3, code: 'Z_DATA_ERROR' }
assert.js:42
  throw new errors.AssertionError({
  ^

AssertionError [ERR_ASSERTION]: undefined == true
    at showProgress (/Users/zfoster/.config/yarn/global/node_modules/now/download/dist/download.js:8723:24)
    at PassThrough.<anonymous> (/Users/zfoster/.config/yarn/global/node_modules/now/download/dist/download.js:2222:49)
    at emitOne (events.js:116:13)
    at PassThrough.emit (events.js:211:7)
    at addChunk (_stream_readable.js:263:12)
    at readableAddChunk (_stream_readable.js:250:11)
    at PassThrough.Readable.push (_stream_readable.js:208:10)

**Comment by mhart** [comment]
Woah, that's weird – that looks like an error with the npm registry, or the way it was packaged or something. As if there's something wrong with the tarball

**Comment by zfoster** [comment]
I think this might actually be a yarn issue

**Comment by southpolesteve** [comment]
Zach, which version of yarn?

**Comment by zfoster** [comment]
1.3.2

**Comment by zfoster** [comment]
not an issue with aws-sudo

**Comment by southpolesteve** [comment]
Dug in a bit with Zach. This is some kind of yarn issue. It works fine for me, but is reproducible on Zach's machine with a fresh install. 

It appears to be attempting to install ALL the current global packages again when this happens. And then breaking in weird ways. An issue for some of the other issues around `yarn global add` https://github.com/yarnpkg/yarn/issues/648

TLDR: Don't use `yarn global`, use `npm -g` instead
